GoogleDataTransportCCTSupport/GDTCCTLibrary/Private/GDTCCTNanopbHelpers.h

```diff
@@ -23,6 +23,14 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/** A string sets in customPrioritizationParams as a key paired to @YES if current event needs to
```

Comment:
*/ on a newline by itself. here and below
```
/** A string sets in customPrioritizationParams as a key paired to @YES if current event needs to
 * populate network connection info data, @NO otherwise.
 */
```